macros/utils/cross_db_utils/current_timestamp.sql

```diff
@@ -1,10 +1,10 @@
 {% macro current_timestamp() -%}
-  {% if dbt_version >= '1.2.0' %}
+  {% if dbt_version >= '1.3.0' %}
         {# This macro is depricated from dbt_utils version 0.9.0, but still hasn't got an equivalent macro at dbt-core #}
         {# Should be replaced to the equivalent macro once it released #}
-        {{ return(dbt_utils.current_timestamp()) }}
+        {{ return(dbt.current_timestamp_backcompat()) }}
     {% else %}
-        {{ return(dbt_utils.current_timestamp()) }}
+        {{ return(dbt.current_timestamp_backcompat()) }}
```

Comment:
My bad, seems like I asked you to change it as well, but it should be:
```suggestion
        {{ return(dbt_utils.current_timestamp()) }}
```